### PR TITLE
Updates to game-service logic

### PR DESCRIPTION
**Pull request by LiamMoMurray**
Added portion of the gameService logic

### CODE REVIEW

File: game-service/src/main/java/com/lms/gameservice/controller/GameController.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ public ResponseEntity<String> joinGame(
             @PathVariable Long gameId) {
 
         // Verify Firebase ID token for user authentication
-        try {
+        try { 
             String uid = authService.validateToken(authorizationHeader);
```
Comment (olanhealy) [inline game-service/src/main/java/com/lms/gameservice/controller/GameController.java:68]: keep spelling as 'authorisation' what has been done before

File: game-service/src/main/java/com/lms/gameservice/controller/GameController.java [hunk 2]
```diff
@@ -82,10 +82,15 @@ public ResponseEntity<String> joinGame(
         }
     }
 
-    @PostMapping("/{gameId}/round/{roundId}/process")
-    public ResponseEntity<String> processRound(@PathVariable Long gameId, @PathVariable Long roundId, @RequestBody List<Long> winningTeamIds) {
+    @PostMapping("/{gameId}/processRound")
+    public ResponseEntity<String> processRound(@PathVariable Long gameId) {
```
Comment (olanhealy) [inline game-service/src/main/java/com/lms/gameservice/controller/GameController.java:86]: Not need to verify logged in? who calls process round? will system call will it be a cron job not really sure

**Review by olanhealy — CHANGES_REQUESTED** [review]
Has this also been tested before pr made? A description on what happens, etc would be needed before approval

**Comment by olanhealy** [comment]
Is this and marks coming in together now? Failing the u
Job atm

**Review by CaoimhinHawk1 — APPROVED** [review]
Looks good to me, approving since its passing unit tests now, good work Liam, get back to Olan on his review and get this pushed ASAP!

**Pull request merged**